packages/live-share-canvas/src/canvas/InkingCanvas.ts

```diff
@@ -319,20 +312,17 @@ export abstract class InkingCanvas {
             if (opacity > 0) {
                 this.canvas.style.opacity = opacity.toString();
 
-                this._additionalLayers.forEach(
-                    (context: CanvasRenderingContext2D) => {
-                        context.canvas.style.opacity = opacity.toString();
-                    }
-                )
+                this._additionalLayers.forEach((context: CanvasRenderingContext2D) => {
```

Comment:
That is less clear than before...

Reply:
what makes this less clear? not sure i can tell

Reply:
avoid parens has been added to re-vote

Reply:
@huntj88 it doesn't make it obvious that first parameter is a callback. You have to scan the full line to get that. If the callback has several parameters, they look like they are the individual parameters of the outer method call. Two ( characters one after the other isn't necessarily something that people can spot immediately, especially, and I'm sorry to use that argument again, when they have my eyes and my age.